patterns/vpc-lattice/cross-cluster-pod-communication/cluster/charts/platform/templates/kyverno-cluster-policy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kyverno.io/v1
```

Comment:
This needs more explanation in the documentation. Its an important detail that is not getting surfaced up properly.

Reply:
I plan to link this also to the associated blog post. I added more info on it but want to keep REadme light